WebRoot/js/zimbraMail/tasks/view/ZmTaskListView.js

```diff
@@ -243,9 +243,8 @@ function(sechdr) {
         htmlArr[idx++] = "<div id='_upComingTaskListHdr'>";
         htmlArr[idx++] = "<table width=100% class='DwtListView-Column'><tr>";
         this.dId = Dwt.getNextId();
-        htmlArr[idx++] = "<td><div class='DwtListHeaderItem-label ";
-        htmlArr[idx++] = ZmTaskListView.SEC_COLOR[sechdr];
```

Comment:
I have kept that so that in future if we need to change the colour for List header item label, it can be referenced.